stdlib/account.go

```diff
@@ -257,6 +259,33 @@ func NewGetAuthAccountFunction(handler AccountHandler) StandardLibraryValue {
 	)
 }
 
+func NewVMGetAuthAccountFunction(handler AccountHandler) StandardLibraryValue {
+	return NewVMStandardLibraryStaticFunction(
+		GetAuthAccountFunctionName,
+		GetAuthAccountFunctionType,
+		getAuthAccountFunctionDocString,
+		func(context *vm.Context, typeArguments []bbq.StaticType, arguments ...interpreter.Value) interpreter.Value {
+			accountAddress, ok := arguments[0].(interpreter.AddressValue)
+			if !ok {
+				panic(errors.NewUnreachableError())
+			}
+
+			referenceType, ok := typeArguments[0].(*interpreter.ReferenceStaticType)
+			if !ok {
+				panic(errors.NewUnreachableError())
+			}
+
+			return NewAccountReferenceValue(
+				context,
+				handler,
+				accountAddress,
+				referenceType.Authorization,
+				interpreter.EmptyLocationRange,
+			)
+		},
```

Comment:
I guess these implementations for the builtin functions were copied over as-is?

Reply:
Yeah, I just moved them over from the vm package, and cross checked they matched the interpreter functions